### ISSUE TITLE
"alarms modal of the dashboard" cannot expand in a custom alarm

### ISSUE THREAD

**Issue by pedro-nonfree** [firehol/netdata#3500]
Hi,

I did a custom alert for hddtemp in `/etc/netdata/health.d/hddtemp.conf`:

```
 $ sudo cat /etc/netdata/health.d/hddtemp.conf 
 alarm: disk_temperature
    on: hddtemp_local.disks_temp
 units: Celsius
 every: 10s
  calc: $collected_total_raw
  crit: $this > 30
  info: bad temperature
    to: sysadmin
```

I cannot spand [the alarms modal of the dashboard](https://github.com/firehol/netdata/issues/2997#issuecomment-343576110)

As the alert is already alerting (as a test) it is not expanded too

![2018-03-02-153207_891x485_scrot](https://user-images.githubusercontent.com/2603422/36903752-e71f6850-1e2e-11e8-8978-02a15a8e93e6.png)

just to show that the metric is working/running

![2018-03-02-153048_1050x336_scrot](https://user-images.githubusercontent.com/2603422/36903708-c85b3cfa-1e2e-11e8-84bc-b062981355ff.png)

**Comment by ktsaou** [comment]
confirmed! this is a bug!

**Comment by ktsaou** [comment]
fixed in PR #3506 
It seems that the alarms modal was not searching for charts by name.
